barretenberg/cpp/src/barretenberg/commitment_schemes/pcs_test_instance_witness_generator.hpp

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include "barretenberg/ecc/curves/bn254/bn254.hpp"
+#include "barretenberg/polynomials/polynomial.hpp"
+#include "barretenberg/transcript/transcript.hpp"
+#include "commitment_key.test.hpp"
+
+namespace bb {
+/**
+ * @brief Constructs random polynomials, computes commitments and corresponding evaluations.
+ *
+ * @tparam Curve
+ */
+template <typename Curve> struct PCSInstanceWitnessGenerator {
```

Comment:
Remove `PCS` in the name. Also I think this could use the Flavor/Settings defined in the other file